test/frontend.js

```diff
@@ -327,7 +324,12 @@ describe('Frontend', function () {
         .click(elT('mailboxLink'))
 
       const alert = await n.exists(elT('alertNotification'))
+
+      console.log('alert:::::::', alert)
```

Comment:
now you can use the amazing new `note` feature for debugging stuff like this

Reply:
Yes, this was before this new era, and I forgot to delete it